src/routes/[...404].tsx

```diff
@@ -0,0 +1,8 @@
+import { useLocation } from "@solidjs/router";
+import { getLocaleFromPathname, missingTranslationMessage } from "~/i18n";
+
+export default function NoEntry() {
```

Comment:
Are we expecting this to be a 404 still? If so should we add a <HttpStatusCode code={404} />?
If not, should we rename this file?

Reply:
If the entry doesn't exist, yes. This should be a `404`. Not adding an error status code, will make crawlers index it.